converter/src/main/java/gov/cms/qpp/conversion/encode/AggregateCountEncoder.java

```diff
@@ -26,6 +27,6 @@ public AggregateCountEncoder(Context context) {
 	@Override
 	protected void internalEncode(JsonWrapper wrapper, Node node) {
 		// simply writes the value in the Node
-		wrapper.putInteger(VALUE, node.getValue("aggregateCount"));
+		wrapper.putInteger(VALUE, node.getValue(AggregateCountDecoder.AGGREGATE_COUNT));
```

Comment:
I'm pretty sure we do this elsewhere, but let's add an issue to the backlog to do a bit of refactoring so that the encoders don't directly reference decoders.